lib/node_modules/@stdlib/stats/base/dstdevyc/docs/types/index.d.ts

```diff
@@ -59,7 +59,7 @@ interface Routine {
 	* var v = dstdevyc.ndarray( x.length, 1, x, 1, 0 );
```

Comment:
Sigh. The lack of decimal points for the `correction` parameter is everywhere in this PR. This was my original mistake.